src/main/scala/org/allenai/plugins/CoreDependencies.scala

```diff
@@ -79,7 +79,7 @@ trait CoreDependencies {
   val sprayCaching = sprayModule("caching")
 
   // Spray json (separate from Spray toolkit)
-  val sprayJson = "io.spray" %% "spray-json" % "1.2.6"
+  val sprayJson = "io.spray" %% "spray-json" % "1.3.1"
```

Comment:
This will be a headache as there are breaking API changes in 1.3. I think we should deal with it though since it provides 10x performance improvement.